pilot/pkg/serviceregistry/kube/controller/ambient/authorization.go

```diff
@@ -432,7 +432,7 @@ const (
 	httpRuleFmt string = "ztunnel does not support HTTP rules (%s require HTTP parsing), in ambient mode you must use waypoint proxy to enforce HTTP rules. %s"
 
 	httpDenyRuleBoilerplate  string = "Deny rules with HTTP attributes will be enforced without their HTTP components. This is more restrictive than intended."
```

Comment:
```suggestion
	httpDenyRuleBoilerplate  string = "DENY policy with HTTP attributes is enforced without the HTTP rules. This will be more restrictive than intended."
```

Reply:
perhaps?  Changes to active voice